core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -497,10 +497,10 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
 
   private def alterConfigsRequest =
     new AlterConfigsRequest.Builder(
-      Collections.singletonMap(new ConfigResource(ConfigResource.Type.TOPIC, tp.topic),
-        new AlterConfigsRequest.Config(Collections.singleton(
-          new AlterConfigsRequest.ConfigEntry(LogConfig.MaxMessageBytesProp, "1000000")
-        ))), true).build()
+        Collections.singletonMap(new ConfigResource(ConfigResource.Type.TOPIC, tp.topic),
+          new AlterConfigsRequest.Config(Collections.singleton(
+            new AlterConfigsRequest.ConfigEntry(LogConfig.MaxMessageBytesProp, "1000000")
+          ))), true).build()
```

Comment:
Can we get rid of whitespace-only changes like this, or at least move them to another PR?

Reply:
Let me check around.